dashboard/config/scripts/play_lab_rocket_6.multi

```diff
@@ -11,4 +11,4 @@ MARKDOWN
 wrong "rocket-height(seconds) = seconds + 12"
 right "rocket-height(seconds) = seconds * 12"
 wrong "rocket-height(seconds) = 48 / seconds"
-wrong "rocket-height(seconds)  = seconds * 48"
\ No newline at end of file
+wrong "rocket-height(seconds) = seconds \* 48"
```

Comment:
remove extraneous `\`? if it is needed, I'm curious why its not also needed on line 12.

Reply:
weird; I'm not sure why this got added, given that it is indeed not needed